contracts/src/v0.8/functions/dev/1_0_0/accessControl/interfaces/ITermsOfServiceAllowList.sol

```diff
@@ -6,6 +6,19 @@ pragma solidity ^0.8.19;
  */
 
 interface ITermsOfServiceAllowList {
+  struct Config {
```

Comment:
Do we actually want to commit to the config being in the public interface? Arguably if we change the config in the future we'll have to break this interface or introduce a new interface. I think dropping all methods related to the config would give us some room to upgrade in the future.

If we do this, please don't forget to push the documentation for these methods down into the actual `TermsOfServiceAllowlist` implementation.